Source/Applications/openXDA/openXDA/openXDA.csproj

```diff
@@ -192,6 +194,10 @@
     <Reference Include="Microsoft.Owin.Security">
       <HintPath>..\..\..\Dependencies\GSF\Microsoft.Owin.Security.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.SqlServer.Types, Version=11.0.0.0, Culture=neutral, PublicKeyToken=89845dcd8080cc91, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>C:\Program Files\Microsoft SQL Server\150\DTS\Extensions\Common\Microsoft.SqlServer.Types.dll</HintPath>
+    </Reference>
```

Comment:
Maybe it has something to do with this. It seems like this should be a NuGet package, but somehow the HintPath is referencing something in Program Files, and the DLL was manually added to `Dependencies\SQL`. Can someone verify that these dependencies are needed and that they're being referenced correctly?